cfglib.py

```diff
@@ -172,6 +172,8 @@
                 help="Don't show ssl warnings"),
     cfg.BoolOpt('keep_affinity_settings', default=False,
                 help="Keep affinity/anti-affinity settings"),
+    cfg.StrOpt('ssh_cipher', default='arcfour128',
```

Comment:
I would recommend either using default value `aes128-ctr,aes192-ctr,aes256-ctr,arcfour256,arcfour128, aes128-gcm@openssh.com,aes256-gcm@openssh.com, aes128-cbc,3des-cbc,blowfish-cbc,cast128-cbc,aes192-cbc, aes256-cbc, arcfour` (taken from `man ssh_config`), or add logic - if `ssh_cipher` option is not set (`None`) do not pass cipher to `ssh` command at all.